### ISSUE TITLE
Ruby error when setting up vagrant32 file on Windows

### ISSUE THREAD

**Issue by fclubb** [LSDtopotools/LSDTT_vagrantfiles#1]
Trying to install vagrant on a Windows Machine, there seems to be a problem with Ruby when running `vagrant up` 

We get the following error:
```
[NOTE]
You may have encountered a bug in the Ruby interpreter or extension libraries.
Bug reports are welcome.
For details: http://www.ruby-lang.org/bugreport.html


This application has requested the Runtime to terminate it in an unusual way.
Please contact the application's support team for more information.
```
From googling, it looks like it might be a problem with one of the vagrant ruby plugins, FFI. 
https://github.com/hashicorp/vagrant/issues/5869
But I was having dependency issues with trying to downgrade FFI from the latest version
I'm going to try downloading vagrant and virtual box and see if I can reproduce the problem, since I don't have access to the windows machine with the problem.

**Comment by fclubb** [comment]
I get the same error on Ubuntu! 
![screenshot from 2017-12-14 11-38-36](https://user-images.githubusercontent.com/12732264/34006304-6eb75646-e0c3-11e7-8a07-b1c55ae82325.png)

**Comment by simon-m-mudd** [comment]
@fclubb can you try uninstalling vagrant 2.01 and installing 2.0?

**Comment by DmitryTa** [comment]
Have the same problem

**Comment by simon-m-mudd** [comment]
One of our users solved this by installing vagrant 2.00. I'm afraid at the moment I can't check this so can someone try to install with virtualbox 5.1 and vagrant 2.0 and report results?

**Comment by fclubb** [comment]
Worked on windows with virtualbox 5.1 and vagrant 2.0.0.
